util.go

```diff
@@ -354,3 +369,22 @@ func fmtTime(u *User, lastStamp time.Time) string {
 	}
 	return lastStamp.In(u.Timezone.Location).Format("3:04")
 }
+
+// See if the public key is there and if it is not, try to create.
+func checkKey(keyPath string) {
+	privateKey, err := rsa.GenerateKey(trueRand.Reader, 4096)
+	if err != nil {
+		return
+	}
+
+	// generate and write private key as PEM
+	privateKeyFile, err := os.Create(keyPath)
+	if err != nil {
+		return
+	}
+	defer privateKeyFile.Close()
+	privateKeyPEM := &pem.Block{Type: "RSA PRIVATE KEY", Bytes: x509.MarshalPKCS1PrivateKey(privateKey)}
```

Comment:
Do you think we should rely on RSA? Should we pick ECDSA instead? What do you think

Reply:
I choose RSA because of how ubiquitous is it. RSA 4096 bits should be safe enough for some times. But if you would prefer to use a new and trendy elliptic curved based algorithm, lets do it.

I have no strong opinion one way or another, it's your call.

Reply:
LOL new and trendy. So true. Yes let’s go for the new and trendy elliptic curve stuff. That whole ssh-rsa-2 thing has made me saltier towards rsa.

Reply:
Good point.